README.md

```diff
@@ -49,4 +49,15 @@ some ticket. Then you have:
  5. Wait for architect to merge (architect may ask additional changes)
  6. After merge to `master` ask issue author to close the ticket
  
+Git guidelines:
+ - start all commit message with tiket number, e.g.: `#1 - some message`
```

Comment:
@g4s8 in cactoos, the rule is `(#1) Some message` and is enforced by gitlint, maybe you would want to adopt one or the other?